application/F3DStarter.cxx

```diff
@@ -1624,7 +1624,24 @@ void F3DStarter::AddCommands()
       const char* file = std::getenv("CTEST_OPEN_DIALOG_FILE");
       if (!file)
```

Comment:
you could consider adding the logic above the if to get coverage for this line, even if its not really tested, at least the code would be executed.